app/src/main/java/ru/vssemikoz/newsfeed/storage/NewsApiRepository.java

```diff
@@ -15,22 +14,15 @@ public NewsApiRepository(MainApplication mainApplication) {
         this.mainApplication = mainApplication;
     }
 
-    public void getNewsFromApi(Category category, Callback<NewsApiResponse> callback, Source source) {
+    public void getNewsFromApi(Category category, Callback<NewsApiResponse> callback) {
         String requestCategory = null;
-        String requestSource = null;
-        String country = "ru";
+        String requestCountry = "ru";
         String KEY = mainApplication.getKEY();
         Call<NewsApiResponse> call;
-
         if (category != Category.ALL) {
             requestCategory = category.name();
```

Comment:
This is not a `requestCategory`. This is a `categoryKey` since we're resolving a key used for that category and then passing it to the request.